examples/blockly-node/index.js

```diff
@@ -23,27 +23,36 @@
 
 var Blockly = require('blockly');
 
-var xmlText = `<xml xmlns="https://developers.google.com/blockly/xml">
-<block type="text_print" x="37" y="63">
-  <value name="TEXT">
-    <shadow type="text">
-      <field name="TEXT">Hello from Blockly!</field>
-    </shadow>
-  </value>
-</block>
-</xml>`;
+var json = {
+  "blocks": {
+    "languageVersion": 0,
+    "blocks": [
+      {
+        "type": "text_print",
+        "id": "0b61B|)zMnTbwbPAG1iG",
+        "x": 113,
+        "y": 238,
+        "inputs": {
+          "TEXT": {
+            "shadow": {
+              "type": "text",
+              "id": "_!OgDm+,dRCHii(i|kdL",
+              "fields": {
+                "TEXT": "Hello from Blockly!"
+              }
+            }
+          }
+        }
+      }
+    ]
+  }
+}
 
 try {
-  var xml = Blockly.Xml.textToDom(xmlText);
-
-  // Create workspace and import the XML
   var workspace = new Blockly.Workspace();
-  Blockly.Xml.domToWorkspace(xml, workspace);
-
-  // Convert code and log output
-  var code = Blockly.Python.workspaceToCode(workspace);
-  console.log(code);
```

Comment:
I think we still want these lines, since they demonstrate converting the blocks you loaded into the workspace into Python code.

Reply:
Yeah, done 🚀